otelcol/command.go

```diff
@@ -63,3 +67,44 @@ func updateSettingsUsingFlags(set *CollectorSettings, flags *flag.FlagSet) error
 	}
 	return nil
 }
+
+func newFeaturesCommand() *cobra.Command {
+	return &cobra.Command{
+		Use:   "features [feature-id]",
+		Short: "Display feature gates information",
+		Long:  "Display information about available feature gates and their status",
+		RunE: func(_ *cobra.Command, args []string) error {
+			if len(args) > 0 {
+				found := false
+				featuregate.GlobalRegistry().VisitAll(func(g *featuregate.Gate) {
+					if g.ID() == args[0] {
+						found = true
+						fmt.Printf("Feature: %s\n", g.ID())
+						fmt.Printf("Enabled: %v\n", g.IsEnabled())
+						fmt.Printf("Stage: %s\n", g.Stage())
+						fmt.Printf("Description: %s\n", g.Description())
+						fmt.Printf("From Version: %s\n", g.FromVersion())
+						if g.ToVersion() != "" {
+							fmt.Printf("To Version: %s\n", g.ToVersion())
+						}
+					}
+				})
```

Comment:
Doesn't block this PR, but this feature is a great use case for https://github.com/open-telemetry/opentelemetry-collector/issues/7625